tests/unit/PageHeader.spec.js

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import BootstrapVue from 'bootstrap-vue';
+import PageHeader from '../../components/PageHeader.vue';
+
+const localVue = createLocalVue();
+localVue.use(BootstrapVue);
+
+const factory = () => shallowMount(PageHeader, {
+  localVue
+});
+
+describe('PageHeader', () => {
+  it('contains a search form with a query field', () => {
+    const wrapper = factory();
+
+    const queryField =  wrapper.find('[data-qa="search query input"]');
+
+    expect(queryField.attributes().name).to.equal('query');
+  });
+
+  it('contains a search form submit button', () => {
```

Comment:
Let's also test that clicking the button results in the function being called.

https://vue-test-utils.vuejs.org/guides/dom-events.html